
From: Eric W. Biederman <ebiederm@xmission.com>

The ppc port sets of init_mm ahead of time to hold an identity mapped page. 
This makes use_mm non-static so they don't have to reinvent the wheel.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/aio.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN fs/aio.c~kexec-use_mm fs/aio.c
--- 25/fs/aio.c~kexec-use_mm	2004-09-21 01:53:50.304275952 -0700
+++ 25-akpm/fs/aio.c	2004-09-21 01:53:50.308275344 -0700
@@ -566,7 +566,7 @@ struct kioctx *lookup_ioctx(unsigned lon
  *	(Note: this routine is intended to be called only
  *	from a kernel thread context)
  */
-static void use_mm(struct mm_struct *mm)
+void use_mm(struct mm_struct *mm)
 {
 	struct mm_struct *active_mm;
 	struct task_struct *tsk = current;
_
